src/lib/VirtualScroller.js

```diff
@@ -425,6 +425,21 @@ class VirtualScroller {
 
         return scrollTop <= offsetTop && offsetTop <= scrollTop + this.containerHeight;
     }
+
+    /**
+     * Gets the currently visible list items
+     * @return {Array<HTMLElement>} - the list of current visible list items
+     */
+    getVisibleItems() {
+        if (!this.listEl) {
+            return [];
+        }
+
+        return Array.prototype.slice
+            .call(this.listEl.children)
+            .filter((itemEl) => this.isVisible(itemEl))
+            .map((itemEl) => itemEl && itemEl.children && itemEl.children[0]);
```

Comment:
Why do we need to look at the first child of the item?

Reply:
From a `VirtualScroller` perspective, the first child of the `itemEl` will be what the user of the `VirtualScroller` generated, or in this case, the generated thumbnail. Basically, it's stripping away the `VirtualScroller` shell and returning just the items

Reply:
👍 , just wanted to make sure it wasn't looking at something thumbnail sidebar specific.